ksp_plugin/plugin.cpp

```diff
@@ -42,6 +43,34 @@ Permutation<WorldSun, AliceSun> const kSunLookingGlass(
 
 }  // namespace
 
+Plugin::Plugin(GUIDToOwnedVessel vessels,
```

Comment:
Why is this body not in its normal position, i.e., after the bodies of all the public methods.  Generally speaking the ordering in this file is a bit of a mess.  Remember that methods should be in the same order in the .h and .cc files.  Please open an issue to reorder the bodies in a separate PR (without any semantically significant changes).